src/component/header.js

```diff
@@ -0,0 +1,312 @@
+  display: flex;
+  flex-direction: row;
+`;
+
+const SocialIcon = Styled(FontAwesome)`
+  color: #222;
+  margin-left: 10px;
+  margin-right: 10px;
+  font-size: 22px;
+`;
+
+const PartnerLogo = Styled.img`
+  margin: 10px;
+  width: 80%;
+  height: 20%;
+`;
+
+const PartnerText = Styled.div`
+  display: flex;
+  flex-direction: row;
+  font-weight: 600;
+  font-size: 14px;
+  margin-right: 20px;
+  text-align: center;
+`;
+
+
+class Header extends React.Component {
+  render() {
+    const { navigationCheck } = this.props;
+    return (
+      <Container id="header">
+        <LogoContainer >
+          <Themeline
+            horizontal
+            alignTop
+          />
+          <LogoTitle>mooniak</LogoTitle>
+          <LogoContent >
+            We are a multydeciplinary design, art direction and technology studio. We provide
+            communication solutions and experience across
+            languages and mediums using graphic design, typography, image-making and
+            digital technologies.
+          </LogoContent>
+          <Themeline
+            horizontal
+            alignBottom
+          />
+        </LogoContainer>
+        <MenuContainer >
+          <CircleBase />
+          <Overlay>
+            <MenuRow>
+              <Column >
+                <Themeline
+                  horizontal
+                  alignTop
+                />
+                <MenuItem>about us</MenuItem>
+              </Column>
+              <Column>
+                <Row>
+                  <MenuItem>colombore</MenuItem>
+                  <Themeline vertical />
+                </Row>
+              </Column>
+            </MenuRow>
+            <MenuRow>
+              <MenuButton
+                href="#services"
+                onClick={ () => navigationCheck('services') }
+              >
+                OUR SERVICES
+              </MenuButton>
+              <MenuButton>TELL US YOUR NEED</MenuButton>
+            </MenuRow>
+            <MenuRow>
+              <Column >
+                <MenuItem
+                  href="#projects"
+                  onClick={ () => navigationCheck('projects') }
+                >projects</MenuItem>
+                <Themeline
+                  horizontal
+                  alignBottom
+                />
+              </Column>
+              <Column>
+                <Row>
+                  <MenuItem>subscribe</MenuItem>
+                </Row>
+              </Column>
+            </MenuRow>
+          </Overlay>
+        </MenuContainer>
+        <PartnersContainer >
+          <Row>
+            <PartnerText>partners</PartnerText>
+            <Themeline
+              horizontal
+              alignTop
+            />
+          </Row>
+          <PartnerLogo src={ Sense } />
+          <PartnerLogo src={ Texual } />
+          <PartnerLogo src={ Scrible } />
+        </PartnersContainer>
+        <ContactContainer >
+          <ContactRow border="solid">
```

Comment:
Add the contact number as a link and add the number in href like this: https://css-tricks.com/the-current-state-of-telephone-links/ Do the same for the email.